src/app.js

```diff
@@ -0,0 +1,65 @@
+import {FilterableTable} from './components/FilterableTable.react';
+
+import * as SharedUtils from './utils/SharedUtils';
+
+document.addEventListener('DOMContentLoaded', function(){
+  var searchObject = SharedUtils.parseUrlSearchString(),
```

Comment:
I'm not sure where to start with this but it can certainly be refactored down into smaller methods